### ISSUE TITLE
sbtc-bridge-web is statically configured

### ISSUE THREAD

**Issue by EmbeddedAndroid** [stacks-archive/sbtc-developer-release#230]
### Expected Behavior

The sbtc-bridge-web app can be configured without source code changes

### Actual Behavior

https://github.com/EmbeddedAndroid/sbtc-bridge-web/commit/2b13c9bd3024f7a965cae042ef57fe82fbdd808f

### Steps to Reproduce

Attempt to use the sbtc-bridge-web interface with leather wallet after a mint.

**Comment by radicleart** [comment]
Will look into and try to reproduce.

**Comment by radicleart** [comment]
@EmbeddedAndroid i just built but the http://stacks-api:3999 service isn't running - i think i saw an open issue for this? 

Another issue is the containers for the web and maybe api are pretty old. Is there a way to remove or override the caching policy for these two containers - main changes regularly and the two apps are coupled?

Aside this there are dependence on [contract deployment](https://github.com/stacks-network/sbtc/pull/238) the bridge won't function until its linked to a contract and also on the [leather on devnet](https://github.com/stacks-network/sbtc/issues/196) issue - i think this is the cause of UTXO related errors as its using the testnet addresses in the wallet to fetch UTXOs.

.

**Comment by radicleart** [comment]
Thanks i can see stacks api now. 

The contract is set on the API via environment parameter in docker compose. Bridge will see it once its deployed.

**Comment by AshtonStephens** [comment]
> Thanks i can see stacks api now.
> 
> The contract is set on the API via environment parameter in docker compose. Bridge will see it once its deployed.

@radicleart I'm going to assign this to you. Are you unblocked on this now?

**Comment by radicleart** [comment]
Hey @AshtonStephens - i am looking at this but don't think the bridge will work on deven until the Leather wallet supports regtest Bitcoin addresses - part of the error above is the bridge trying to fetch UTXOs from regtest using the testnet addresses in the wallet. 

In meantime I've prioritised trying to run Romeo on testnet against the Bridge.

**Comment by radicleart** [comment]
Working to fix this  and stumbled on this issue https://github.com/getumbrel/umbrel-os/issues/248 that affects the bridge on devnet. the mempool /:address/utxo, that the api relies on, is not implemented in mempool devnet (its only supported in testnet/mainnet).

Looking for workarounds for this.
